v2/manager.go

```diff
@@ -326,6 +326,22 @@ func (c *Manager) Stat() (*stats.Metrics, error) {
 		ThpFaultAlloc:         out["thp_fault_alloc"].(uint64),
 		ThpCollapseAlloc:      out["thp_collapse_alloc"].(uint64),
 	}
+	cur_mem := getStatFileContent(filepath.Join(c.path, "memory.current"))
+	max_mem := getStatFileContent(filepath.Join(c.path, "memory.max"))
+	cur_swap := getStatFileContent(filepath.Join(c.path, "memory.swap.current"))
+	max_swap := getStatFileContent(filepath.Join(c.path, "memory.swap.max"))
+
+	usage := stats.MemoryEntry{
+		Usage: cur_mem,
```

Comment:
try not to use snake case `_` in the go code.

Reply:
oh, yeah, sorry, Python legacy;)